MANIFEST.in

```diff
@@ -14,7 +12,4 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-# Sets up the Python environment for development under .venv
-
-rm -f Pipfile
-PIPENV_VENV_IN_PROJECT=1 pipenv --python="3.7" install -r py_dev_requirements.txt
+graft netanalysis/third_party
```

Comment:
Maybe add a comment on what this does. I think people are not familiar with `MANIFEST.in`

Reply:
Done in 9d6eac5

Reply:
Also, this is a very weird diff